app/views/grant_submissions/submissions/award/update.turbo_stream.haml

```diff
@@ -0,0 +1,5 @@
+- user_grant_role = current_user.get_role_by_grant(grant: @grant)
```

Comment:
user_grant_role might be something we would want to pass in the controller instead of querying in the view, esp since its redefined a few times on other pages.